src/index.js

```diff
@@ -42,3 +42,4 @@ export {
     Avatar,
     Icon
 }
+
```

Comment:
No need to `import` just to `export`. You can just `export X from './components/X`. Also it would be nice to sort the export alphabetically

Reply:
Good to know, I will give that a try. Thank you very much! 😊


My initial reasoning was to be able to do `import { X } from '@doist/reactist'` and wasn't sure if that works without an index.js exporting all components. Especially when it comes to duplicate names (e.g. two components exporting an `Item` as sub components)

Reply:
@henningmu I started this review before realizing sitting down with @jvalente and go through the code would be way more productive, trying to get that to happen :)